VERSION

```diff
@@ -1 +1 @@
-7.0.0
+7.164.8
```

Comment:
Consider 7.164.0, to support the detection of snapshot versions (cf https://github.com/vespa-engine/system-test/pull/474)

Reply:
Fixed